driver-core/src/main/java/com/datastax/driver/core/ReplicationStategy.java

```diff
@@ -55,6 +56,7 @@ static ReplicationStrategy create(Map<String, String> replicationOptions) {
             }
         } catch (NumberFormatException e) {
             // Cassandra wouldn't let that pass in the first place so this really should never happen
+            logger.error("Failed to parse replication options: " + replicationOptions, e);
```

Comment:
Did you manage to actually reproduce a `NumberFormatException` here? It is obviously possible to reproduce this error artificially, i.e. by invoking this method directly with invalid parameters, but is it ever triggered when dealing with real keyspaces? I tried with different C* versions and replication strategies, but I invariably get back an `InvalidConfigurationInQueryException` if the replication factor is not numeric, so the exception is never raised here. 
If you _did_ manage to trigger this error somehow, I would suggest rephrasing the comment  above as it seems inaccurate (and if possible, describe a situation where such an error can happen).
Otherwise, given that `NumberFormatException` is an unchecked exception and does not require to be caught, I would suggest that we simply get rid of this catch block altogether.